src/Sentry/Laravel/Tracing/EventHandler.php

```diff
@@ -176,6 +184,12 @@ protected function queryExecutedHandler(DatabaseEvents\QueryExecuted $query): vo
             }
         }
 
+        if ($this->recordSqlBindings) {
+            $context->setData(array_merge($context->getData(), [
+                'db.sql.bindings' => $query->bindings
+            ]));
+        }
```

Comment:
Can you move this above `traceSqlQueryOrigins`, so we have the same order?

Reply:
I've gone ahead and made those changes 👍